AzureDataLake/Scripts/ProcessCCB.usql

```diff
@@ -1,51 +1,106 @@
 ﻿SET @@FeaturePreviews = "DataPartitionedOutput:on";
 
-USE DATABASE [DecisionServiceDBTest];
+USE DATABASE [DecisionServiceDB];
 
-REFERENCE ASSEMBLY DecisionServiceDBTest.[DecisionServiceExtractor];
+REFERENCE ASSEMBLY DecisionServiceDB.[DecisionServiceExtractor];
+
+DECLARE @OutputSlimLogs bool = true;     //true if you want to have slime version of logs (all metadata without features)
 
 DECLARE @_appId string = "";    //your appId
 DECLARE @_account string = "";    // your account name
 
-DECLARE @_start DateTime = new DateTime(2019, 10, 1);  //your inclusive start date
-DECLARE @_end DateTime = new DateTime(2019, 10, 12);    //your non-inclusive end date
+DECLARE @_start DateTime = new DateTime(2019, 10, 25);  //your inclusive start date
+DECLARE @_end DateTime = new DateTime(2019, 10, 30);    //your non-inclusive end date
 
 DECLARE @OutputFolder string = "/daily/" + @_appId + "/";      //your output folder
      
 DECLARE @InputFile string = "wasb://"+ @_appId +"@" + @_account + ".blob.core.windows.net/{model}/data/{date:yyyy}/{date:MM}/{date:dd}_{filename}.json";
 
-DECLARE @OutputFileStatistics string = @OutputFolder + "statistics-{date:yyyy}-{date:MM}-{date:dd}.csv";
+DECLARE @OutputFileStatisticsCB string = @OutputFolder + "statistics-cb-{date:yyyy}-{date:MM}-{date:dd}.csv";
```

Comment:
What is the meaning of the CB postfix here? Why are these stats CB? Do we have other stats that are CCB?